app-config.yaml

```diff
@@ -1,6 +1,7 @@
 app:
   title: Backstage Example App
   baseUrl: http://localhost:7000
+  googleAnalyticsTrackingId: # UA-9393929-4
```

Comment:
Let's make the example something like `UA-00000000-0`

Reply:
Done